File: 2022/01/js/2.js
const fs = require("fs");
let data = fs.readFileSync("i.txt").toString();
let p = data.split(/\r?\n\r?\n/)
let r = p.map(r => r.split(/\r?\n/).reduce((acc, n) => acc + parseInt(n, 10), 0)).sort((a, b) => b - a).slice(0, 3).reduce((acc, n) => acc + n, 0);
console.log(r);

File: 2022/01/py/2.py
with open("i.txt") as f:
    data = [l.removesuffix("\n") for l in f]

c = []
s = 0
for l in data:
    if l == "":
        c.append(s)
        s = 0
    else:
        s += int(l)

c.sort(reverse=True)

print(c[0]+c[1]+c[2])